independent-publisher-connectors/Google Gemini/readme.md

```diff
@@ -0,0 +1,122 @@
+# Google Gemini
+
+## Publisher: Priyaranjan KS, Vidya Sagar Alti [Tata Consultancy Services]
+
+## Overview
+Custom connector for Google Gemini, providing advanced AI text generation functionalities.
+
+## Prerequisites
+An API key from Google is required to access the Google Gemini API.
+
+### Obtaining an API Key
```

Comment:
Please change this to Obtaining Credentials

Reply:
@vmanoharas Sure, Will do that. Mean time, I am doing one more round of regression testing, will keep you posted in a day or two. Thanks.

Reply:
@vmanoharas The swagger and readme updates are completed.